core/src/main/java/apoc/cypher/CypherInitializer.java

```diff
@@ -70,6 +88,15 @@ public void available() {
                     } catch (Exception ignored) {
                         userLog.info("Cannot check APOC version compatibility because of a transient error. Retrying your request at a later time may succeed");
                     }
+
+                    // this block is already in system db, so we can directly execute system operation
+                    // for backward compatibility: clear system nodes at start-up if the corresponding database name doesn't exist
+                    // placed here instead of e.g. `ApocConfig` because at this point we are sure that all databases are recovered
+                    cleanUpSystemNodes();
```

Comment:
I don't think this is true. Databases are brought up in an eventual consistent way in 5.x so when system is up neo4j could not be up

Reply:
Yes indeed it is not safe.
I've been looking for other ways to do this but I don't think there is currently a method that isn't completely error prone, but maybe I'm missing something.

Perhaps it's better not to do this thing at all and document somewhere that for backward compatibility we have to delete the triggers manually. Wdyt?